Riot/Managers/Call/CallPresenter.swift

```diff
@@ -455,11 +455,11 @@ class CallPresenter: NSObject {
         #if canImport(JitsiMeetSDK)
         JMCallKitProxy.addListener(self)
         
-        guard let session = sessions.first else {
+        guard let sessionInfo = sessions.first else {
```

Comment:
I'm wondering if this renaming in `CallPresenter` was intentional, or caught up by mistake?

Reply:
Xcode's `rename` feature ....